brand-guidelines/color-principles.md

```diff
@@ -24,7 +24,7 @@ A minimal palette, clear hierarchy, good information design, and ample whitespac
 
 </div>
 
-<h2 id="palette">Palette<span class="cf-code-link"><a href="https://github.com/cfpb/cf-colors/blob/gh-pages/src/cf-colors.less">View code <i class="cf-icon cf-icon-external-link"></i></a></span></h2>
+<h2 id="palette">Palette<span class="cf-code-link"><a href="https://github.com/cfpb/generator-cf/blob/master/app/templates/src/static/css/brand-palette.less">View code <i class="cf-icon cf-icon-external-link"></i></a></span></h2>
```

Comment:
Please update this link to https://github.com/cfpb/cf-theme-cfpb/blob/master/src/color-palette.less. Even if cf-theme-cfpb doesn't take hold, I really don't want the generator to be considered the canonical location of the palette.